### PR TITLE
[Misc] Avoid making the buttons sticky when they are not displayed

### PR DESCRIPTION
**Pull request by atallahade**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js [hunk 1]
```diff
@@ -384,11 +384,12 @@ require(['jquery'], function ($) {
 
   $(window).on("scroll resize load click xwiki:dom:refresh", function() {
     var isFullScreen = $('.fullScreenWrapper').length > 0
+    var isVisible = $container.css('display') != 'none';
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js]: Why not https://api.jquery.com/visible-selector/ ?

    var isVisible = $container.is(':visible');

**Comment by mflorea** [comment]
Can you give an example where this is currently happening? (i.e. when the action buttons are hidden)

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js (hunk 1 above)
Comment (atallahade) [reply xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/actionbuttons/actionButtons.js]: It's even better, thanks.

**Comment by atallahade** [comment]
@mflorea 

When you create / edit an AWM:
sticky-bug

I'm only scrolling down on the video.

**Pull request merged**